docs/docs/guides/build-workflows.md

```diff
@@ -98,13 +98,22 @@ example.CHAIN_CRONTAB:
     - B
     - C
   periodic:
-    schedule: "0 */3 * * *"
+    crontab: "0 */3 * * *"
 ```
 
-The `periodic > schedule` key takes an integer (unity is the second) or a string argument
-([crontab](https://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#crontab-schedules)
-syntax). So in the first example, the *example.CHAIN* worflow will be executed **every 60 seconds**
-and the second one, *example.CHAIN_CRONTAB*, **every three hours**.
+The format is the following (the [official documentation](https://docs.celeryproject.org/en/v4.4.7/userguide/periodic-tasks.html#crontab-schedules) of the `crontab` function gives some examples of each attribute):
+
+```yaml
+periodic:
+  crontab: "minute hour day_of_month month_of_year day_of_week"
+```
+
+So in the first example, the *example.CHAIN* workflow will be executed **every 60 seconds** and the second one, *example.CHAIN_CRONTAB*, **every three hours**.
+
+!!! warning
+    Older versions of Celery Director used the `periodic > schedule` key which is now deprecated.
+
+    It is strongly advised to migrate to `periodic > interval` or `periodic > crontab` keys to set up a scheduled workflow.
```

Comment:
* To migrate from schedule to interval: if your schedule is in seconds just rename `schedule` to `interval`
* To migrate from schedule to crontab: if your schedule uses the cron syntax rename `schedule` to `crontab`. There is also a fix when migrating to this syntax: the old `schedule` syntax incorrectly parses the cron string as `minute hour day_of_week day_of_month month_of_year` but the new `crontab` syntax properly parses the cron string as `minute hour day_of_month month_of_year day_of_week`. So be careful to this inversion when migrating here.

Reply:
Done